systemtests/src/main/java/io/kroxylicious/Constants.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious;
```

Comment:
We should probably use a sub-package for systemtest code.

Reply:
done